### PR TITLE
Fix multiple decoding issues

### PR DESCRIPTION
**Pull request by DJBen**
name-list, headers, formatted-text etc. Reducing LilyPondTest suite failures to ~~73~~ 69.

### CODE REVIEW

File: Sources/MusicXML/Simple Types/WedgeType.swift [hunk 1]
```diff
@@ -11,7 +11,7 @@
 /// single wedge is divided into multiple segments.
 public enum WedgeType: String {
     case crescendo
-    case descrescendo
```
Comment (jsbean) [inline Sources/MusicXML/Simple Types/WedgeType.swift:14]: 🤦‍♂

File: Tests/MusicXMLTests/Complex Types/DirectionTests.swift [hunk 2]
```diff
@@ -46,4 +46,55 @@ class DirectionTests: XCTestCase {
         )
         XCTAssertEqual(decoded, expected)
     }
+
+    func decodingDirection_wedge() throws {
```
Comment (jsbean) [inline Tests/MusicXMLTests/Complex Types/DirectionTests.swift]: I think these need to be named `testDecodingDirection_wedge()` or similar in order for `XCTest` to run them?

File: Tests/MusicXMLTests/Complex Types/DirectionTests.swift [hunk 3]
```diff
@@ -46,4 +46,55 @@ class DirectionTests: XCTestCase {
...
+        let expected: Direction = Direction(
+            placement: .above,
+            directionType: .wedge(
+                Wedge(type: .crescendo, spread: 0)
+            )
+        )
+        XCTAssertEqual(decoded, expected)
+    }
+
+    func decodingDirection_octaveShift() throws {
```
Comment (jsbean) [inline Tests/MusicXMLTests/Complex Types/DirectionTests.swift]: See above.

File: Tests/MusicXMLTests/Complex Types/DirectionTests.swift [hunk 4]
```diff
@@ -46,4 +46,55 @@ class DirectionTests: XCTestCase {
...
+        </direction>
+        """
+        let decoded = try XMLDecoder().decode(Direction.self, from: xml.data(using: .utf8)!)
+        let expected: Direction = Direction(
+            directionType: .octaveShift(OctaveShift(type: .down, size: 8))
+        )
+        XCTAssertEqual(decoded, expected)
+    }
+
+    func decodingDirection_bracket() throws {
```
Comment (jsbean) [inline Tests/MusicXMLTests/Complex Types/DirectionTests.swift]: Same.

File: Tests/MusicXMLTests/Complex Types/DirectionTests.swift (hunk 2 above)
Comment (DJBen) [reply Tests/MusicXMLTests/Complex Types/DirectionTests.swift]: oh damn

**Comment by DJBen** [comment]
Amended.

**Comment by jsbean** [comment]
Bada bing, bada boom.

**Pull request merged**